vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -968,7 +968,9 @@ std::vector<AdInfo> AdsImpl::GetEligibleAds(
     const std::vector<AdInfo>& ads) {
   std::vector<AdInfo> eligible_ads = {};
 
-  for (const auto& ad : ads) {
+  auto unseen_ads = GetUnseedAdsAndRoundRobinIfNeeded(ads);
```

Comment:
Typo in method name (should be `Unseen`)?

Reply:
thanks sir :-), all fixed